clients/gtest/gemm_gtest.cpp

```diff
@@ -87,6 +87,12 @@ namespace
         static std::string name_suffix(const Arguments& arg)
         {
             RocBLAS_TestName<gemm_test_template> name;
+
+            if(strstr(arg.function, "_bad_arg") != nullptr)
+            {
+                name << "_bad_arg";
```

Comment:
if (strstr(arg.function, "_bad_arg"))

Reply:
Yeah good point. This also doesn't print quite as I intended (pre_checkin/gemm.blas3/_bad_argf16_r_NN_n1_n1_n1_1_n1_n1_0_n1). Just added it to run the added bad_arg check. Will fix or delay for a more fitting PR.